cmds/status/status.go

```diff
@@ -144,13 +138,18 @@ func ReadCachedURLsFile(path string) (cachedURLs *CachedURLs, err error) {
 // Cache writes the pingURLs p to a file at path (replacing if it exists
 // already, and creating parent folders, if required), using the current time
 // for the retrieval timestamp.
-func (p PingURLs) Cache(path string) (cachedURLs *CachedURLs, err error) {
-	color.Magenta("Writing cache file %v", path)
-	parentDir := filepath.Dir(path)
+func (p PingURLs) Cache(cache *configdir.Config, cachePath string) (cachedURLs *CachedURLs, err error) {
+	color.Magenta("Writing cache file %v", filepath.Join(cache.Path, cachePath))
+
+	///////////////////////////////////////////////////////////////////////
+	// workaround until https://github.com/shibukawa/configdir/pull/2 lands
```

Comment:
Please add a github issue in this repo to remove this workaround when the time is right.